config/artifact-caching-proxy_azure.yaml

```
@@ -11,3 +11,6 @@ ingress:
 
 persistence:
   storageClass: managed-csi-premium-retain
+
+nodeSelector:
+  agentpool: arm64small
```

```suggestion

nodeSelector:
  agentpool: x86medium
```

Switching ACP to ARM looks risky for now, as the Nginx tuning is making assumption that might not be the same in ARM kernel + 4vCPUs on the underlying machine is not enough